src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php

```diff
@@ -11,20 +11,20 @@
 
 namespace Sulu\Component\Rest\Tests\Unit\ListBuilder;
 
-use Doctrine\Common\Persistence\ObjectManager;
 use Sulu\Component\Rest\ListBuilder\ListRestHelper;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 
 class ListRestHelperTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @var ObjectManager
+     * @var RequestStack|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
Yes, PhpStorm recognize then all methods of the mock object and of the `RequestStack`.